tsconfig.json

```diff
@@ -15,7 +15,9 @@
     "isolatedModules": true,
     "noEmit": true,
     "jsx": "react-jsx",
-    "noImplicitAny": false
+    "noImplicitAny": false,
+    "typeRoots": ["src/types/custom_types", "node_modules/@types"]
+    // "typeRoots": ["./src/types/custom_types", "./node_modules/@types"]
```

Comment:
Cool, let's hope nothing breaks for other people 😆 nothing seemed to be wrong on my machine with this change.